internal/server/server.go

```diff
@@ -0,0 +1,32 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/arturzhamaliyev/Flight-Bookings-API/internal/config"
+	"github.com/arturzhamaliyev/Flight-Bookings-API/internal/server/handler"
+	"github.com/arturzhamaliyev/Flight-Bookings-API/internal/service"
+	"github.com/gin-gonic/gin"
+)
+
+// New will instantiate a new instance of Server.
+func New(cfg config.Config, services service.Service) *http.Server {
+	h := handler.New(services)
+
+	r := gin.Default()
+
+	r.GET("/health", h.HealthCheck)
+
+	v1 := r.Group("/v1")
+
+	users := v1.Group("/users")
+	{
+		users.POST("/", h.CreateUser)
+	}
```

Comment:
routing isn't related to the server, what if I want run here graphql server?

Reply:
please add description for comment

Reply:
for example:
I want to create another server for accepting http/graphql/gRPC connections but with the current implementation I will get only another http server with provided routes what is not good for the codebase flexibility